tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,363 @@
+        assertionThrown,
+        'testOptions was a function'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: testSeries
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        !assertionThrown,
+        'testOptions was an Object'
+    );
+});
+
+test( '"Series" property needs to be an array', function( assert ) {
+
+    let assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: null
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was empty'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: "string"
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was string'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: undefined
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was undefined'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: true
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was a boolean'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: 666
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was number'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: function(){}
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was a function'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: testSeries
+        });
+    } catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        !assertionThrown,
+        'testSeries was an Array'
+    );
+});
+
+test( "Style method returns correct string", function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries
+    });
+
+    assert.equal( component.get( 'style' ).string, 'height: auto; width: auto;' );
+
+    component.set( 'height', 10 );
+
+    assert.equal( component.get( 'style' ).string, 'height: 10; width: auto;' );
+});
+
+test( "Title property is set", function( assert ) {
```

Comment:
The test for the `title` property shouldn't be that it can be set on the component and read off of it for two reasons: 1) that is expected Ember functionality that is covered by Ember tests and 2) any property can be set and read this way on a component, so what makes the `title` property special?  Only if we're doing something or reacting to a property that makes it worthy of testing.  In the case of `title` there is no component logic associated with it, but there is template logic, which is what this test should cover.